packages/turbo-repository/__tests__/find.test.ts

```diff
@@ -29,4 +37,54 @@ describe("Workspace", () => {
       "packages/ui": ["apps/app"],
     });
   });
+
+  describe("affectedPackages", () => {
+    const tests: AffectedPackagesTestParams[] = [
+      {
+        description: "app change",
+        files: ["apps/app/file.txt"],
+        expected: [{ name: "app-a", relativePath: "apps/app" }],
+      },
+      {
+        description: "lib change",
+        files: ["packages/ui/a.txt"],
+        expected: [
+          // { name: "app-a", relativePath: "apps/app" }, // TODO: this should be included
```

Comment:
A little confused by the TODO, either this is a pretty large bug that should get fixed before merging or the API expects the caller to build up the closure of dependents and then this is the correct behavior.

Reply:
the TODO is there because we _want_ this behavior, but turbo core doesn't implement it. I can remove the expectation since it isn't clear